plugin/environment.go

```diff
@@ -86,9 +86,11 @@ func scanSearchPath(path string) ([]*model.BundleInfo, error) {
 		if !file.IsDir() || file.Name()[0] == '.' {
 			continue
 		}
-		if info := model.BundleInfoForPath(filepath.Join(path, file.Name())); info.ManifestPath != "" {
+		info := model.BundleInfoForPath(filepath.Join(path, file.Name()))
+		if info.Manifest != nil && info.Path != "" {
```

Comment:
It's not obvious why we test `info.Path` here -- `BundleInfoForPath` constructs it from the `path` we provide, which necessarily can't be empty here. Should we just test `info.Manifest != nil`?

Reply:
Makes sense, the check was there already and TBH I don't have much context here so I've left it :)